### PR TITLE
add asset/details and storm/{reboot,stop,start}

### PR DESCRIPTION
**Pull request by sgsullivan**

### CODE REVIEW

**Comment by sgsullivan** [comment]
draft atm because more support could be needed

**Comment by sgsullivan** [comment]
this is ready for review, as I would like to get a release out with these changes as its needed for another project. Any additional support that is required will be in another PR.

File: storm/server_client.go [hunk 1]
```diff
@@ -82,9 +71,45 @@ func (c *ServerClient) Status(id string) (*ServerStatus, error) {
...
+func (c *ServerClient) Start(uniqId string) (*ServerStart, error) {
+	var result ServerStart
+	args := map[string]interface{}{
+		"uniq_id": uniqId,
+	}
+
+	err := c.Backend.CallIntoInterface("bleed/server/start", args, &result)
 
-	return &result, nil
+	return &result, err
```
Comment (dwburke) [inline storm/server_client.go]: is it not safer to return nil for the result if there's an error?  I assume you can't trust the data in `result` if there's an error, but it's possible someone might ignore the error

Comment (sgsullivan) [reply storm/server_client.go]: changed it to that method 3d637bc34b8ddc1875c90539f334d0fddeefe698

**Review by dwburke — APPROVED** [review]

**Pull request merged**